Tools/data/include_utils.h

```diff
@@ -26,10 +26,10 @@ namespace DataUtil
 using AGS::Common::HError;
 using AGS::Common::String;
 
-// Pass a list of files that is filtered in place by ignore-like patterns in a file
-HError IgnoreFiles(std::vector<String> &files, String parent, String ignore_file, bool verbose);
+// Pass a list of files that is filtered in place by include-like patterns in a file
+HError IncludeFiles(std::vector<String> &files, const String& parent, const String& include_pattern_file, bool verbose);
```

Comment:
Please, make this function have separate arguments for input and output files vector. IMHO that's more convenient because there may be different situations, and also easier to understand that this function requires *input files* (I did not understand this first, and though that this function also tests the directory somehow). The caller may replace source vector with resulting one if they need to.